hwdb.d/60-evdev.hwdb

```diff
@@ -733,7 +733,7 @@ evdev:input:b0003v258Ap001E*
  EVDEV_ABS_35=::15
  EVDEV_ABS_36=::15
 
-#########################################
+th#########################################
```

Comment:
What??

Reply:
I used the online editor. Do not know how this happened.